src/functions-and-arrays.js

```diff
@@ -1,21 +1,46 @@
+const arr = require("jshint/data/non-ascii-identifier-start");
+
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(n1, n2) {
+  if (n1>n2){
+    return n1;
+    } else{
+      return n2;
+    }
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
 
+  let largeWord= words[0];
+  let largestWord;
 
+for (let i = 0; i<words.length; i++){
 
-// Iteration #3: Calculate the sum
-const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+  if (words[i].length>largeWord.length) {
+    largestWord=strings[i];
+  }
+}
+
+}
+
```

Comment:
- Aquí te falta cubrir algunos corner cases que vienen en los test, como que por ejemplo devuelva null cuando se introduce un array vacío, lo que se solucionaría con un 
`if(words.length === 0) { return null }`
- Además, la función no retorna nada, te dejo como lo habría hecho yo para que puedas comparar (además intentas asignar `largestWord` a `strings[i]`, pero no tienes ninguna variable llamada `strings`):

```suggestion
function findLongestWord(words) {

  if (words.length === 0) { return null }

  let largestWord = ""

  for (let i = 0; i < words.length; i++) {
    if (words[i].length > largestWord.length) {
      largestWord = words[i];
    }
  }

  return largestWord;

}
```